hazenlib/__init__.py

```diff
@@ -75,11 +75,11 @@
 Welcome to the Hazen Command Line Interface
 Usage:
     hazen <task> <folder> [--measured_slice_width=<mm>] [--report] [--calc_t1 | --calc_t2] [--plate_number=<n>] [--show_template_fit]
-    [--show_relax_fits] [--show_rois]
+    [--show_relax_fits] [--show_rois] [--verbose]
     hazen -h|--help
     hazen -v|--version
 Options:
-    <task>    snr | slice_position | slice_width | spatial_resolution | uniformity | ghosting | relaxometry
+    <task>    snr | slice_position | slice_width | spatial_resolution | uniformity | ghosting | relaxometry | snr_map
```

Comment:
Thanks for your detailed feedback. 

I developed this routine before I knew about Hazen, and bolted it on afterwards--hence there are some minor differences in algorithm and implementation. For example, this routine uses non-parametric phantom recognition (thresholding), whereas `snr`  fits a circle or rectangle to the signal region. I considered merging with the SNR task, but didn't want to change these details in the existing implementation, so just copied the SNR code I'd written before starting on Hazen.

I don't know if this is best and would value community discussion on whether tweaking the current SNR task is acceptable--this would likely cause minor variations in reported SNR values.

Reply:
@tomaroberts does this tie in with what you're working on? I remember you said you were looking at a different method for defining the signal region?

Reply:
I've just seen this! Sorry I missed it till now.

Does sound similar to threshold-based method I implemented in `snr`. I think given that Paul has done this is an entirely separate function, we should proceed. Consolidating the methods or picking one is something to keep in mind for the future though to avoid code duplication, but don't think really a problem at this stage.